dna_rna.py

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+
+import sys
+
+for line in sys.stdin:
+    line = line.rstrip()
+    
+class Rna(self):
+    def _init_(self, line):
+        self.line = line
+        if len(line) < 0:
+            raise ('lil line')
+        elif TypeError:
+            raise TypeError('not a line')
+    def gc(self, line):
+        c_count = 0
+        g_count = 0
+        for i in line:
+            if i == "G":
+                g_count =+ 1
+            elif i == "C":
+                c_count =+ 1
+        return ((g_count + c_count)/len(line))*100        
+    def reverse_complement(self, line):
+        complement_seq = ''
+        nucleotides = {'A': 'U', 'G': 'C', 'C': 'G', 'T': 'A'}

```

Comment:
If you want to use this method for RNA you should add U:A

Reply:
Done!